integrations/sonarqube/client.py

```diff
@@ -338,12 +350,20 @@ async def get_issues_by_component(
         component_key = component.get("key")
         endpoint_path = ""
 
-        if self.is_onpremise:
+        current_version = await self._get_sonarqube_instance_version()
+
+        if current_version >= MINIMUM_SUPPORTED_ISSUES_LIST_VERSION:
```

Comment:
Can you verify if this logic will work for SonarCloud too? From the way I see it, they don't have issues/list, only issues/search

https://sonarcloud.io/web_api/api/issues/search?deprecated=false